src/qt/splashscreen.cpp

```diff
@@ -30,23 +30,24 @@
 SplashScreen::SplashScreen(const NetworkStyle* networkStyle)
     : QWidget(), curAlignment(0)
 {
-    // set reference point, paddings
-    int paddingRight            = 50;
-    int paddingTop              = 50;
-    int titleVersionVSpace      = 17;
-    int titleCopyrightVSpace    = 40;
 
     float fontFactor            = 1.0;
     float devicePixelRatio      = 1.0;
+
+    // set reference point, paddings
     devicePixelRatio = static_cast<QGuiApplication*>(QCoreApplication::instance())->devicePixelRatio();
+    int paddingRight            = 180*devicePixelRatio - 100;
+    int paddingTop              = 180*devicePixelRatio + 0;
 
     // define text to place
-    QString titleText       = PACKAGE_NAME;
-    QString versionText     = QString("Version %1").arg(QString::fromStdString(FormatFullVersion()));
-    QString copyrightText   = QString::fromUtf8(CopyrightHolders(strprintf("\xc2\xA9 %u-%u ", 2009, COPYRIGHT_YEAR)).c_str());
-    QString titleAddText    = networkStyle->getTitleAddText();
+    QString titleText           = PACKAGE_NAME;
+    QString packageNameSubStr   = titleText.mid(0,7);
+    QString splashText          = QString("Connecting to The %1 Network").arg(packageNameSubStr);
```

Comment:
```suggestion
    QString splashText          = QString("Connecting to the %1 network").arg(packageNameSubStr);
```
`the` and `network` should not be capitalized

Reply:
I just kinda like "title case" better - since "The Bitcoin Network" is an entity unto itself. :)
You can define it as a collection of sovereign individuals - and in this way - I believe title case is appropriate.
But I will change it...